archivers/bzip3/Portfile

```diff
@@ -17,4 +18,8 @@ checksums           rmd160  3940e049433a348ccc9fe4f0fcc3fe76cbae37ad \
                     sha256  bba90d867f53efa4291de1df9c22da1578e1a93ca819e0fc68881da6fcc4149d \
                     size    399621
 
+# https://trac.macports.org/ticket/66644
+compiler.blacklist-append \
+                    *gcc-4.* {clang < 400}
```

Comment:
@jmroot From the error message I thought it was not working correctly in C99. Should we change that to the flag?

Reply:
The error in the ticket is complaining about using a C99 feature outside of C99 mode (because the older compilers default to `-std=gnu89`). Yes, it would be better to add a flag than to add a compiler dependency.

Reply:
@jmroot Ok, I will test and update then.